modules/configuration/help/subproject.md

```diff
@@ -0,0 +1,9 @@
+# Configuration: Subproject
+
+Use this page to manage the configuration of existing subprojects, or to add a new one. Your subprojects are listed on the left side of the scren. Click on any subproject name to edit its settings, and click **Save.**
+
+Click **New SubprojectID** to create a new subproject. Populate the fields and click **Save**. Your new subproject should now appear in the list after you refresh the page.
+
+Note: Clicking save more than once will register a duplicate SubprojectID.  If you create an extra ID, you have to delete it from the database manually with an sql command.
```

Comment:
@ridz1208 Do you know whether this is true? If so it sounds like a pretty basic SQL bug that we should fix.

Reply:
no longer the case, this is an old bug